lib/hutch/worker.rb

```diff
@@ -20,6 +21,7 @@ def initialize(broker, consumers)
     # never returns.
     def run
       setup_queues
+      setup_procs.each(&:call)
```

Comment:
Because the worker sets up the queues, I figured that the other setup stuff should go here as well.